pkg/util/users/users.go

```diff
@@ -124,6 +138,30 @@ func (strategy DefaultUserSyncStrategy) alterPgUserSet(user spec.PgUser, db *sql
 	return nil
 }
 
+func (strategy DefaultUserSyncStrategy) alterPgUserRename(user spec.PgUser, db *sql.DB) error {
+	var query string
+
+	renamedBack := false
+
+	nameSuffixDiff := len(user.Name) - len(constants.RoleRenameSuffix)
+
+	if nameSuffixDiff > 0 {
+		if user.Name[nameSuffixDiff:] == constants.RoleRenameSuffix {
```

Comment:
imo the usage of standard functions makes the code clearer:
```golang
strings.HasSuffix(user.Name, constants.RoleRenameSuffix)
```
and
```golang
newName := strings.TrimSuffix(user.Name, constants.RoleRenameSuffix)
```
so no need for `nameSuffixDiff`

Reply:
ah, good to know :smiley: